apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/controller/OnboardingController.java

```diff
@@ -419,6 +419,26 @@ public Uni<Response> update(@PathParam(value = "onboardingId") String onboarding
                 .map(ignore -> Response.status(HttpStatus.SC_NO_CONTENT).build());
     }
 
+    @Operation(
+            summary = "Update recipient code",
+            description = "Update recipient code receiving onboarding id.",
+            operationId = "updateOnboardingRecipientIdUsingPUT"
+    )
+    @PUT
+    @Tag(name = "support")
```

Comment:
Was it requested by Support for Support External API?

Reply:
@manuraf will be replaced by the new api-group